### DISCUSSION TITLE
Command to reset progress in towny

### DISCUSSION THREAD

**Discussion by eat-your-glory** [TownyAdvanced/Towny#6844] [Q&A] [answered]
Hello, I’d need a command to reset all progress in the Towny plugin without resetting whole config file.
Is there anyway to do this?

**Comment by LlmDl** [comment, marked as answer]
By progress do you mean all of the towns and nations and claimed land?

If so, just delete the `\towny\data\` folder.

**Comment by eat-your-glory** [comment]
Alright! Thank you so much for helping!

Il giorno sab 15 lug 2023 alle 15:31 LlmDl ***@***.***> ha
scritto:
